src/lazy/encoding.rs

```diff
@@ -54,20 +85,9 @@ impl<'data> TextEncoding<'data> for TextEncoding_1_1 {
     }
 }
 
-/// The Ion 1.0 binary encoding.
```

Comment:
🗺️ I moved things around in this file to make read a bit better from top to bottom. Traits and their impls are grouped throughout.